Source/cielim/Private/RenderingFunctionsLibrary.h

```diff
@@ -22,14 +22,13 @@ class  CIELIM_API URenderingFunctionsLibrary :
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
 	static void ApplyPSF_Gaussian();
 
-	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
-	static void WriteImage();
-
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
 	static void ApplyCosmicRays();
 
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
 	static void ApplyReadNoise();
 
-	static void ReadImage(){};
+	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
+	static FString ApplyDarkCurrentNoise(FString Filepath, double MaxSigma, double MinSigma, FVector SunPosition, FVector SpacecraftPosition, FVector SpacecraftDirection);
```

Comment:
🔶 Because this line extends over 120 characters, you can drop each parameter onto a new line.

Reply:
Put each parameter onto a new line!